examples/analog_hamiltonian_simulation/06_Analog_Hamiltonian_simulation_with_PennyLane.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    counts_aws = program_aws(detuning_para)
</pre><p>I think <code>param</code>  is a more recognizable abbreviation of <code>parameter</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
bump

Reply:
Sorry I missed this the first time. I have changed `detuning_para` to `detuning_param` and `amplitude_para` to `amplitude_param`.